.github/workflows/release-request.yml

```
@@ -0,0 +1,137 @@
+
+      - name: Create release branch and commit
+        run: |
+          VERSION=${{ steps.cz.outputs.version }}
+          BRANCH_NAME="release-v$VERSION"
+          git checkout -b $BRANCH_NAME
+          git add .
+          git commit -m "bump: release $VERSION"
+          git push origin $BRANCH_NAME
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)